setup.py

```diff
@@ -30,19 +30,19 @@
         raise
 
 # configure the env
-libraries = ['SDL', 'SDL_mixer']
+libraries = ['SDL2', 'SDL2_mixer']
 library_dirs = []
-include_dirs = ['/usr/include/SDL']
+include_dirs = ['/usr/include/SDL2', '/home/kivy/.buildozer/android/platform/android-ndk-r9c/sources/android/support/include']
 extra_objects = []
 extra_compile_args =['-ggdb', '-O2']
 extra_link_args = []
 extensions = []
 
 if not have_cython:
-    libraries = ['sdl', 'sdl_mixer']
+    libraries = ['SDL2', 'SDL2_mixer']
 else:
     include_dirs.append('.')
-    include_dirs.append('/usr/include/SDL')
+    include_dirs.append('/usr/include/SDL2')
```

Comment:
It seems to me that this part is linux only (I might be wrong and it's macos too), so I'd probably move it down to the conditions under either `elif platform == 'linux'` or rather just under `else` because it's `sys.platform`. Otherwise I think we end up with this state anyway e.g. if on iOS:

`include_dirs`:

    [
        '/usr/include/SDL2',  # most likely unused
        '/home/kivy/.buildozer/android/platform/android-ndk-r9c/sources/android/support/include',  # bad
        '/usr/include/SDL2',  # duplicate
        join(kivy_ios_root, 'build', 'include'),  # actually used include
        join(kivy_ios_root, 'build', 'include', 'SDL2'),  # actually used include
    ]